app/functions.js

```diff
@@ -31,9 +31,9 @@ module.exports = function (env) {
     return moment().format('YYYY-MM-DD')
   }
 
-  // Get current month as "word" (eg December = 'December'), or number (eg December = 12)
+  // Get current month as "nameOfMonth" (eg December = 'December'), or number (eg December = 12)
   functions.currentMonth = (type) => {
```

Comment:
Should this have a default, making the param optional?

```suggestion
  functions.currentMonth = (type="number") => {
```